scripts/ci/pylint_todo.txt

```diff
@@ -241,3 +242,91 @@
+./tests/contrib/operators/test_emr_create_job_flow_operator.py
+./tests/contrib/operators/test_s3_copy_object_operator.py
+./tests/contrib/operators/test_s3_delete_objects_operator.py
+./tests/contrib/operators/test_sftp_operator.py
+./tests/providers/snowflake/operators/test_snowflake.py
+./tests/providers/snowflake/operators/test_s3_to_snowflake.py
+./tests/contrib/operators/test_ssh_operator.py
+./tests/contrib/sensors/test_bash_sensor.py
+./tests/contrib/sensors/test_emr_step_sensor.py
+./tests/contrib/sensors/test_file_sensor.py
+./tests/contrib/sensors/test_python_sensor.py
+./tests/contrib/sensors/test_redis_pub_sub_sensor.py
+./tests/contrib/sensors/test_weekday_sensor.py
+./tests/test_core.py
+./tests/executors/test_celery_executor.py
+./tests/executors/test_dask_executor.py
+./tests/hooks/test_docker_hook.py
+./tests/hooks/test_hive_hook.py
+./tests/hooks/test_http_hook.py
+./tests/hooks/test_oracle_hook.py
+./tests/hooks/test_pig_hook.py
+./tests/hooks/test_postgres_hook.py
+./tests/hooks/test_webhdfs_hook.py
+./tests/jobs/test_backfill_job.py
+./tests/jobs/test_base_job.py
+./tests/jobs/test_scheduler_job.py
+./tests/lineage/backend/test_atlas.py
+./tests/lineage/test_lineage.py
+./tests/macros/test_hive.py
+./tests/models/test_cleartasks.py
+./tests/models/test_dag.py
+./tests/models/test_dagbag.py
+./tests/models/test_dagrun.py
+./tests/models/test_pool.py
+./tests/models/test_taskinstance.py
+./tests/operators/test_bash_operator.py
+./tests/operators/test_cassandra_to_gcs.py
+./tests/operators/test_gcs_to_s3.py
+./tests/operators/test_hive_operator.py
+./tests/operators/test_operators.py
+./tests/operators/test_papermill_operator.py
+./tests/operators/test_python_operator.py
+./tests/operators/test_s3_file_transform_operator.py
+./tests/operators/test_s3_to_redshift_operator.py
+./tests/operators/test_virtualenv_operator.py
+./tests/plugins/test_plugin.py
+./tests/security/test_kerberos.py
+./tests/sensors/test_base_sensor.py
+./tests/sensors/test_external_task_sensor.py
+./tests/sensors/test_http_sensor.py
+./tests/sensors/test_named_hive_partition_sensor.py
+./tests/sensors/test_s3_key_sensor.py
+./tests/sensors/test_s3_prefix_sensor.py
+./tests/sensors/test_sql_sensor.py
+./tests/sensors/test_timedelta_sensor.py
+./tests/sensors/test_timeout_sensor.py
+./tests/task/task_runner/test_standard_task_runner.py
+./tests/test_configuration.py
+./tests/test_local_settings.py
+./tests/test_logging_config.py
+./tests/test_utils/reset_warning_registry.py
+./tests/ti_deps/deps/fake_models.py
+./tests/ti_deps/deps/test_ready_to_reschedule_dep.py
+./tests/utils/log/test_es_task_handler.py
+./tests/utils/log/test_file_processor_handler.py
+./tests/utils/log/test_s3_task_handler.py
+./tests/utils/test_cli_util.py
+./tests/utils/test_compression.py
+./tests/utils/test_dag_processing.py
+./tests/utils/test_db.py
+./tests/utils/test_decorators.py
+./tests/utils/test_helpers.py
+./tests/utils/test_log_handlers.py
+./tests/utils/test_tests.py
```

Comment:
I am wondering why all those files got added.

Reply:
This shouldn't be right.

Reply:
fixed by rebasing master